src/components/ButtonPanel.js

```diff
@@ -1,39 +1,46 @@
 import React from 'react';
+import propTypes from 'prop-types';
 import Button from './Button';
 
-function ButtonPanel() {
+function ButtonPanel(props) {
+  const { clickHandler } = props;
   return (
     <>
       <div className="button-row">
-        <Button buttonName="AC" color="grey" />
-        <Button buttonName="+/-" color="grey" />
-        <Button buttonName="%" color="grey" />
-        <Button buttonName="÷" />
+        <Button buttonName="AC" color="grey" handleClick={clickHandler} />
+        <Button buttonName="+/-" color="grey" handleClick={clickHandler} />
+        <Button buttonName="%" color="grey" handleClick={clickHandler} />
+        <Button buttonName="÷" handleClick={clickHandler} />
       </div>
       <div className="button-row">
-        <Button buttonName="7" color="grey" />
-        <Button buttonName="8" color="grey" />
-        <Button buttonName="9" color="grey" />
-        <Button buttonName="x" />
+        <Button buttonName="7" color="grey" handleClick={clickHandler} />
+        <Button buttonName="8" color="grey" handleClick={clickHandler} />
+        <Button buttonName="9" color="grey" handleClick={clickHandler} />
+        <Button buttonName="x" handleClick={clickHandler} />
       </div>
       <div className="button-row">
-        <Button buttonName="4" color="grey" />
-        <Button buttonName="5" color="grey" />
-        <Button buttonName="6" color="grey" />
-        <Button buttonName="-" />
+        <Button buttonName="4" color="grey" handleClick={clickHandler} />
+        <Button buttonName="5" color="grey" handleClick={clickHandler} />
+        <Button buttonName="6" color="grey" handleClick={clickHandler} />
+        <Button buttonName="-" handleClick={clickHandler} />
       </div>
       <div className="button-row">
-        <Button buttonName="1" color="grey" />
-        <Button buttonName="2" color="grey" />
-        <Button buttonName="3" color="grey" />
-        <Button buttonName="+" />
+        <Button buttonName="1" color="grey" handleClick={clickHandler} />
+        <Button buttonName="2" color="grey" handleClick={clickHandler} />
+        <Button buttonName="3" color="grey" handleClick={clickHandler} />
+        <Button buttonName="+" handleClick={clickHandler} />
       </div>
       <div className="button-row">
-        <Button buttonName="0" color="grey" wide="true" />
-        <Button buttonName="." color="grey" />
-        <Button buttonName="=" />
+        <Button buttonName="0" color="grey" wide handleClick={clickHandler} />
+        <Button buttonName="." color="grey" handleClick={clickHandler} />
+        <Button buttonName="=" handleClick={clickHandler} />
       </div>
```

Comment:
- [ ] passe `handleClick` as `clickHandler` props to each `Button` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/panel/passToButton.png))